### DISCUSSION TITLE
May I know when will the leaderboard be fixed？

### DISCUSSION THREAD

**Discussion by ZiwenYeee** [PaddlePaddle/PaddleSpatial#76] [BaiduKDDCup2022] [answered]
Some questions about leaderboard should be fixed:

1. Missing value in submission. In the evaluation script, if submission contains missing values, the score of missing value part will be counted as zero. Does it mean that all the score in the public leaderboard is wrong? If so, can we get the right score for all the past submissions or earn more chances to submit?

2. It seems that test_y can be assessed by envs["path_to_test_y"], should it be hidden?

By the way, the current fake score from my submission is generated by the gru baseline, is it possible to fix it?

**Comment by ramber1836** [comment]
Hi, there,

1. The false high ranks have been updated partially and it is still in
progress.
2. test_y is located at different locations compared to test_x and the
evaluation script, so as to the submitted zip file.

Thanks for your kind advise.


Regards,
Xinjiang Lu

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

The online tests have been updated to avoid the false high scores, please refer to the GitHub(https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation) updates for more details.

The leaderboard is also updated
